### ISSUE TITLE
OSError: [Errno 101] Network is unreachable when listening on *down* interfaces

### ISSUE THREAD

**Issue by michaelarnauts** [jstasiak/python-zeroconf#25]
I have multiple network interfaces (a bridge, subinterfaces on eth0, an ipv6 tunnel and openvpn). I assume this causes the following issue:

```
$ python3 test.py
Press enter to exit...

Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib/python3.4/threading.py", line 920, in _bootstrap_inner
    self.run()
  File "/home/michael/.local/lib/python3.4/site-packages/zeroconf.py", line 1057, in run
    self.zc.send(out)
  File "/home/michael/.local/lib/python3.4/site-packages/zeroconf.py", line 1657, in send
    bytes_sent = s.sendto(packet, 0, (addr, port))
OSError: [Errno 101] Network is unreachable
```

When I specify my main IP address in `ZeroConf(["192.168.1.254"])`, it works fine. 

I guess it should silently ignore interfaces that can't be used (because they have no IP maybe?) instead of stopping completly.

**Comment by michaelarnauts** [comment]
It seems that the library doesn't check if the interface is up or down. It only checks if the interface has an IP address.

The issue can be reproduced as follows (on linux):
```
# brctl addbr test0
# ifconfig test0 172.16.20.1 netmask 255.255.255.0
# ifconfig test0 down
```

**Comment by stephenrauch** [comment]
This appears to be a duplicate of #19

**Comment by stephenrauch** [comment]
Closed via #72
